### ISSUE TITLE
Translator and managers should be able to flag comments and suggestions as abuse

### ISSUE THREAD

**Issue by flodolo** [mozilla/pontoon#2928]
Occasionally we see users coming in and spamming comments, or leaving random suggestions (I found 3 this morning, after the Norwegian localizer flagged one). In the worst case scenario, they're not just random but offensive. 

There's no way for real users to flag these for admin review, and it's very hard to clean up after them.

1. Translator and managers should be able to flag them, e.g with a REPORT ABUSE command that sends an email.
2. Admins should be able to easily see the user's recent activity (translations, comments). Maybe that should be part of the email, since doing it through Django admin is a huge pain.
3. Admins should be able to easily remove the entire user's activity (translations, comments).

The reason for limiting this to translators and managers is to avoid abuse of the abuse report. And maybe we don't send the email if the user has already been reported (flag in the user profile?).

**Comment by mathjazz** [comment]
> * Admins should be able to easily see the user's recent activity (translations, comments). Maybe that should be part of the email, since doing it through Django admin is a huge pain.

Would including comments on the profile page help with that?

> * Admins should be able to easily remove the entire user's activity (translations, comments).

Should there be an option to delete abusive users the old-school way (i.e. without preserving data we care about)?

**Comment by flodolo** [comment]
> Would including comments on the profile page help with that?

Not really, because the admin would need to click every link to check what they wrote (like I just had to do for translations in case of one user).

> Should there be an option to delete abusive users the old-school way (i.e. without preserving data we care about)?

I actually prefer to have these users disabled instead of removed, so they won't sign up again (although we need #2896).

**Comment by mathjazz** [comment]
> > Would including comments on the profile page help with that?
> 
> Not really, because the admin would need to click every link to check what they wrote (like I just had to do for translations in case of one user).

Wouldn't the ability to reveal comments / translations on the Profile page itself be generally useful (even outside the context of this issue)?

> > Should there be an option to delete abusive users the old-school way (i.e. without preserving data we care about)?
> 
> I actually prefer to have these users disabled instead of removed, so they won't sign up again (although we need #2896).

Right. Translation and comments would be deleted, though, not disabled, right?

**Comment by flodolo** [comment]
> Wouldn't the ability to reveal comments / translations on the Profile page itself be generally useful (even outside the context of this issue)?

Yes. I thought you were talking about adding links to comments, like we currently have.

> Translation and comments would be deleted, though, not disabled, right?

Correct, completely deleted.
